### PR TITLE
feat: add solana send usd example app

### PR DESCRIPTION
**Pull request by keyvankhademi**

### CODE REVIEW

**Review by guibescos — APPROVED** [review]

File: target_chains/solana/examples/send_usd/app/package.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
+  "name": "app",
```
Comment (guibescos) [inline target_chains/solana/examples/send_usd/app/package.json:2]: We should add this to lerna like the evm example

File: target_chains/solana/examples/send_usd/app/src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+      await transactionBuilder.buildVersionedTransactions({
+        computeUnitPriceMicroLamports: 50000,
+      }),
+      { skipPreflight: true }
+    );
+  }
+}
+
+function Button() {
+  const connection = new Connection("http://devnet.xyz.pyth.network");
```
Comment (guibescos) [inline target_chains/solana/examples/send_usd/app/src/App.tsx:95]: `https://api.devnet.solana.com`

File: target_chains/solana/examples/send_usd/app/src/App.tsx [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    const priceServiceConnection = new PriceServiceConnection(
+      "https://hermes.pyth.network/",
+      { priceFeedRequestConfig: { binary: true } }
+    );
+    const pythSolanaReceiver = new PythSolanaReceiver({
+      connection,
+      wallet: wallet as Wallet,
+    });
+
+    const priceUpdateData = await priceServiceConnection.getVaa(
```
Comment (guibescos) [inline target_chains/solana/examples/send_usd/app/src/App.tsx:48]: Can you replace this by 
`const priceUpdateData = await priceServiceConnection.getLatestVaas([SOL_PRICE_FEED_ID])`

File: target_chains/solana/examples/send_usd/app/package.json (hunk 1 above)
Comment (keyvankhademi) [reply target_chains/solana/examples/send_usd/app/package.json:2]: Done.

File: target_chains/solana/examples/send_usd/app/src/App.tsx (hunk 2 above)
Comment (keyvankhademi) [reply target_chains/solana/examples/send_usd/app/src/App.tsx:95]: Done.

File: target_chains/solana/examples/send_usd/app/src/App.tsx (hunk 3 above)
Comment (keyvankhademi) [reply target_chains/solana/examples/send_usd/app/src/App.tsx:48]: Done.

**Pull request merged**